### PR TITLE
Final School website

### PR DESCRIPTION
**Pull request by killy10o10**
This is a responsive website for a school. Images are added to suit the theme of a kid's school. The teacher section of the home page is generated dynamically in javascript and the mobile screen menu feature is also implemented.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,9 +1,5 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (roshan-bajgain) [inline README.MD]: - [x]  Kindly write a brief description of the project in the about section. Currently, users who visit your repo will not have a good picture of what the project is about.

File: README.MD [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
 
-# Portfolio Project
-
-
-
 ## Live Demo
```
Comment (roshan-bajgain) [inline README.MD]: - [x] There is no live demo so I would suggest you add live demo so any user who visit your repo can see your website easily.

File: README.MD [hunk 3]
```diff
@@ -36,6 +32,7 @@ Contributions, issues, and feature requests are welcome!
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
+> Image Credits to. (istockphoto.com) 
 
 - Hat tip to anyone who's code/resource was used
 - Inspiration
```
Comment (roshan-bajgain) [inline README.MD]: - [x] In your README file kindly provide the `Acknowledgement` that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: about.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+          <li><a href=""></a><i class="fa-brands fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+      <nav class="desktop-nav">
+        <a href="index.html" class="nav-logo" ><img src="./images/school-logo.png" alt=""></a>
+        <ul class="flex-list">
+          <li><a href="about.html">About</a></li>
```
Comment (roshan-bajgain) [inline about.html]: - [x] There should be possible to navigate between the pages, there's a link from the `Home` page to the `About` page and the opposite too. When I see there is a link for the about page only but when I try to navigate to the `Home page` There is no link so kindly fix it.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+          <li><a href="#"></a>Program</li>
+          <li><a href="#"></a>Sponsor</li>
+          <li><a href="#"></a>News</li>
+          <li class="last-child bubbly-button"><a href="#"></a>Killy Int'l</li>
+        </ul>
+      </nav>
```
Comment (roshan-bajgain) [inline about.html]: - [ ] Your design is not like the design provided by microverse. your logo should be aligned a little bit right but it is totally on the left and the `a tag` should have the same color, font size, and font weight. Kindly adjust it
### your design
his design
### Behance design
figam

File: about.html [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    <section id="about-school">
+      <h2 id="about-heading">About School</h2>
+      <div class="about-intro">
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam iste
+          ducimus, officiis dicta eligendi ad beatae non itaque corrupti alias
+          natus quo saepe sit fugiat at repellendus consequatur. Corrupti, ex.
+        </p>
+      </div>
+      <div class="logo">
```
Comment (roshan-bajgain) [inline about.html:71]: - [x] Your pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design. So kindly make changes
### your design
his home
### Behance design
figma

File: index.html [hunk 7]
```diff
@@ -1,17 +1,160 @@
...
+      <button type="button" id="join-button" class="bubbly-button">
+        Register Your Child
+      </button>
+    </section>
+    <!--Teachers Section  -->
+    <section id="teacher-section">
+      <h2 class="section-intro">Our Teachers</h2>
+      <div id="teacher-grid">
+         
+      </div>
```
Comment (roshan-bajgain) [inline index.html]: - [x] you are missing this small image beside the image so kindly fix it
### Original design
hhh

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @killy10o10, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- [x]  I cannot see any video recording link about the project either in your PR summary or the readme file. Kindly record video documentation about the project and provide the link either in your PR summary or your readme. 🙏

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+      <nav class="desktop-nav-2">
+        <ul class="flex-list">
+          <li><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+          <li><a href=""></a><i class="fa-brands fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+      <nav class="desktop-nav">
```
Comment (gtekle) [inline about.html:32]: - [ Optional ] Great, that you have done the changes required by the previous reviewer on the navbar. However, I believe you can still do some improvements to make it look like in the design. 
#### Design
image

#### Impl
image

File: about.html [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+          <li><a href="index.html">Register</a></li>
+          <li><a href="#about-school">Mission</a></li>
+          <li><a href="#testimonials">Facts</a></li>
+          <li class="last-child bubbly-button"><a href="#"></a>Killy Int'l</li>
+        </ul>
+      </nav>
+    </header>
+    <!-- Headline Section -->
+    <section id="headline-section">
+      <h1 class="intro-heading">LEARN THE PRACTICAL WAY</h1>
```
Comment (gtekle) [inline about.html:45]: - [x] As pointed out by the previous reviewer, one of the project requirements is to stick to the design as much as possible. In this case, the main font color used is #EC5242 with some background color clipping effect. Besided, I believe the text less accessible in its current form. what do you think?
image
#### Design
image
#### Impl
image

File: about.html [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+      </div>
+      <div class="logo">
+        <img src="./images/school-logo.png" alt="" />
+      </div>
+    </section>
+    <!--Standout Section  -->
+    <section id="standout-section">
+      <h2 class="section-intro">Let Your Kid Standout</h2>
+      <hr class="orange-hr" />
+    </section>
```
Comment (gtekle) [inline about.html:79]: - [ Optional ] Well done on the cool animation effect in the `Register your child` button. And can we add the hover effect on the program card (Stats card in your case) by adding an `outline` as in the Design?
#### Design
image
#### Impl
image

File: index.html [hunk 11]
```diff
@@ -1,17 +1,160 @@
...
+    <!--Teachers Section  -->
+    <section id="teacher-section">
+      <h2 class="section-intro" id="teacher-heading">Our Teachers</h2>
+      <div id="teacher-grid">
+         
+      </div>
+
+      <button type="button" id="view-more" class="bubbly-button">
+        MORE <i class="fa-solid fa-chevron-down orange"></i>
+      </button>
```
Comment (gtekle) [inline index.html:141]: - [ Optional ] In the design, the `More` button in the mobile version was supposed to render more Teachers when clicked. By default it renders only two speakers(teachers in your case). What do you think implementing the JavaScript logic to make the button functional. You may also add `Less` to collapse it back to two teachers.
image

File: main.js [hunk 12]
```diff
@@ -0,0 +1,79 @@
+//  ==CLOSE BUTTON FOR MENU===//
+const hamburger = document.getElementById('hamburger');
```
Comment (gtekle) [inline main.js:2]: - [ Optional] I really like the nice mobile nav you created. However, could you make it cover the whole screen and stop the body from scrolling?
image

File: about.html [hunk 13]
```diff
@@ -0,0 +1,114 @@
...
+        </div>
+        <!-- Fact Card Two -->
+        <div class="fact-card" id="fact-two">
+          <h2>2005</h2>
+          <p>Best child education center</p>
+        </div>
+      </div>
+    </section>
+    <!--Footer -->
+    <footer id="footer">
```
Comment (gtekle) [inline about.html:101]: - [ Optional ]  The footer in the desktop version about page has the logo on the left column and the description text on the right in one row.
#### Design
image
#### Impl
image

- [ Optional ] In mobile version the layout remains the same as the desktop, but the background color become `white`. And the partner's sectoin becomes visible as well.
#### Footer About page - Mobile
image

File: README.MD [hunk 14]
```diff
@@ -38,8 +41,9 @@ Give a ⭐️ if you like this project!
 ## Acknowledgments
 
 - Hat tip to anyone who's code/resource was used
-- Inspiration
-- etc
+- Image Credits: (https://www.istockphoto.com)
+- Cindy Shin: [@adgio07](https://www.behance.net/adagio07)
```
Comment (gtekle) [inline README.MD]: - [x] It is good that you acknowledged the Cindy. However, you can use `Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07)` instead to indicate that the original design idea is by Cindy.
image
#### Appropriate attribution
image

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @killy10o10 :wave: , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
this-close


# Highlights
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Nice UI design
:heavy_check_mark: Clean code
:heavy_check_mark: No linter errors

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @killy10o10   , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Decriptive readme file.
:heavy_check_mark: Well implemented design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.MD [hunk 15]
```diff
@@ -1,32 +1,35 @@
...
 
+> This is a project created using HTML, CSS and JavaScript to show my knowledge of them.
 
 ## Live Demo
+[Live Demo Link](https://killy10o10.github.io/HTML-CSS-JavaScript-capstone-project/)
 
-
-> HTML/CSS & JavaScript capstone project.
+## Video Presentation
+[Video Demo Link](https://www.loom.com/share/f2970e1a4e854e41b80d8ba64a92be63)
```
Comment (mahbubul14) [inline README.MD]: - [x] Your video presentation is excellent but you missed adding "a highlight about a piece of the code the student finds interesting or is proud of making". Kindly add that part to your video. :+1:

File: about.html (hunk 13 above)
Comment (mahbubul14) [inline about.html:101]: - [ ] In the footer, it should contain a logo aligned to the left and text aligned to the right. Kindly check the following design and work on it for the homepage and the about page both. :+1: 

**Note:** 
- [ ] For the mobile version, the "About page footer background will be white not dark" :+1: 

Your design for Hompage |  Your design for about page
:-------------------------:|:-------------------------:
f1 | f3

Required design for Hompage |  Required design for about page
:-------------------------:|:-------------------------:
f2 | f4

File: index.html [hunk 16]
```diff
@@ -1,17 +1,160 @@
...
+          <li><a href="#teacher-section">Teachers</a></li>
+          <li><a href="#sponsors">Sponsors</a></li>
+          <li class="last-child bubbly-button"><a href="#"></a>Killy Int'l</li>
+        </ul>
+      </nav>
+    </header>
+    <!-- Headline Section -->
+    <section id="headline-section">
+      <h1 class="intro-heading">JOYFUL LEARNING</h1>
+      <h1 class="intro-heading">KILLY INTERNATIONAL</h1>
```
Comment (mahbubul14) [inline index.html:46]: - [ ] It is better for your page to have just one h1 tag, this makes your website semantic and easily readable by browsers. :+1: 

- [ ] I think you should add a small line of text on the top of the page title. :+1: 

Your Implementation  | Design
:-------------------------:|:-------------------------:
h2 | h1

File: about.html (hunk 13 above)
Comment (mahbubul14) [inline about.html:101]: - [ ] The partner section is missing from the about page. Kindly make it visible on the about us page.

Your Implementation             |  Design
:-------------------------:|:-------------------------:
p1 | p2

File: index.html (hunk 11 above)
Comment (mahbubul14) [inline index.html:141]: - [ ] The "more button" should be increased in size and try to match the design.

Your implementation   |  Design
:-------------------------:|:-------------------------:
m1 | m2

- [ ] **[ Optional ]** In the design, the More button in the mobile version was supposed to render more Teachers when clicked. By default, it renders only two speakers(teachers in your case). What do you think about implementing the JavaScript logic to make the button functional? You may also add Less to collapse it back to two teachers.

File: styles/button.scss [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, transparent 10%, $button-bg 15%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%);
+    background-size: 10% 10%, 20% 20%, 15% 15%, 20% 20%, 18% 18%, 10% 10%, 15% 15%, 10% 10%, 18% 18%;
+    //background-position: 0% 80%, -5% 20%, 10% 40%, 20% 0%, 30% 30%, 22% 50%, 50% 50%, 65% 20%, 85% 30%;
+  }
```
Comment (mahbubul14) [inline styles/button.scss]: - [ ] Kindly remove the inline comment from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history.

File: styles/button.scss [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, transparent 10%, $button-bg 15%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%),
+      radial-gradient(circle, $button-bg 20%, transparent 20%);
+    background-size: 15% 15%, 20% 20%, 18% 18%, 20% 20%, 15% 15%, 10% 10%, 20% 20%;
+    //background-position: 5% 90%, 10% 90%, 10% 90%, 15% 90%, 25% 90%, 25% 90%, 40% 90%, 55% 90%, 70% 90%;
+  }
```
Comment (mahbubul14) [inline styles/button.scss]: - [ ] Kindly remove the inline comment from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history.

File: styles/style.css [hunk 19]
```diff
@@ -0,0 +1,421 @@
...
+  justify-content: center;
+  align-items: center;
+  padding: 20px;
+  background-color: #272a31;
+  color: #8d8d8d;
+}
+
+#sponsors-container {
+  /* display:grid;
+  grid-template-columns: repeat(3, auto); */
```
Comment (mahbubul14) [inline styles/style.css]: - [ ] Kindly remove the inline comments from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history.

File: styles/style.css [hunk 20]
```diff
@@ -0,0 +1,421 @@
...
+#sponsors-container *:hover {
+  filter: grayscale(0%) contrast(100%);
+}
+
+/* Footer */
+footer {
+  display: flex;
+  justify-content: center;
+  padding: 20px;/* align-items: center; */
+}
```
Comment (mahbubul14) [inline styles/style.css]: - [ ] Kindly remove the inline comment from your code. The old pieces of code as inline comments make your project look messy. If you need to review a previous version of your code, you can always use git history.

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @killy10o10 ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] In your `README` file, there is no content for the "Getting Started" section. Kindly add content for the getting started section. This should be an example of how you may give instructions on setting up your project locally.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linters error.
:heavy_check_mark: Good Readme.
:heavy_check_mark: Clean code.

### Optional suggestions

- [ ] Kindly check the inline comments under the review. :arrow_down: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.MD (hunk 15 above)
Comment (killy10o10) [reply README.MD]: @mahbul14 Thanks for the review. But I did highlight the special feature in my video which is the bubbly button I implemented to suit the theme of a kid school.

**Review by MahmoudBakr23 — COMMENTED** [review]
Hi @killy10o10 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done working on the required changes from the previous review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**